### DISCUSSION TITLE
AirTag

### DISCUSSION THREAD

**Discussion by Knutnoh** [rednblkx/HomeKey-ESP32#107] [Ideas]
Would it be possible to read the Apple AirTag in order to trigger automations? I don't give my children an iPhone yet because they are too young, but an Airtag would make sense

**Comment by rednblkx** [comment]
If you use Home Assistant, any nfc tag you scan will be available in the Tags section which can be used in automations, other than that you can use raw MQTT and extract the uid from “auth topic” and use that in automations with node-red or whatever else you have.

**Comment by fisch55** [comment]
I just tried to hold my airtrag to the ESP with Homekey.... and at the same time listened to mqtt, but nothing arrived. I guess you mean that with another reader and not with your ESP HomKey project, right?

**Reply by Knutnoh** [reply]
> I just tried to hold my airtrag to the ESP with Homekey.... and at the same time listened to mqtt, but nothing arrived. I guess you mean that with another reader and not with your ESP HomKey project, right?

At HomeAssistant it will generate a new Tag for the AirTag. Mqtt also doesn't work for me

**Reply by fisch55** [reply]
Sure ? With this project esp Homekey?

Useing this Project- but no AirTag is listed in ha ….

**Reply by fisch55** [reply]
EDIT: IT WORKS !!!! SORRy
